docs/tutorials/gestures.rst

```diff
@@ -74,17 +76,18 @@ easy to turn into a program::
             sleep(1000)
             display.scroll(random.choice(answers))
 
-Most of the program is a list called ``answers``. The actual game is in the
-``while`` loop at the end.
+Większość programu to tablica nazwana ``answers`` (ang. odpowiedzi). 
+Sama gra znajduje się w pętli ``while`` na końcu.
 
-The default state of the game is to show the character ``"8"``. However, the
-program needs to detect if it has been shaken. The ``was_gesture`` method uses
-its argument (in this case, the string ``"shake"`` because we want to detect
-a shake) to return a ``True`` / ``False`` response. If the device was shaken
-the ``if`` conditional drops into its block of code where it clears the screen,
-waits for a second (so the device appears to be thinking about your question)
-and displays a randomly chosen answer.
+Domyślny stan gry wyświetla cyfrę ``"8"``. Jednakże, program musi wykryć, 
+że nastąpiło potrząśnięnie. Metoda ``was_gesture`` używa swojego argumentu (w tym przypadku
+ciągu ``"shake"`` ponieważ chcemy wykryć potrząśnięcie) i zwraca ``True`` lub ``False``.
+
+Jeżeli urządzenie było potrząśnięte, warunek ``if`` wykonuje swoj blok kodu, w którym 
+czyści ekran, czeka przez sekundę (aby urządzenie wyglądało, jakby zastanawiało się nad 
+twoim pytaniem), a następnie wyświetla losowo wybraną odpowiedź.
+
+Dlaczego by nie spytać, czy jest to najlepszy program kiedykolwiek napisany? Co mógłbyś
```

Comment:
Nie wiem czemu, ale nie podoba mi się to zdanie — jest jakieś sztywne. Może pomińmy całą pierwszą część i zostawmy tylko "Czyż nie jest to najlepszy program na świecie?"